Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -195,7 +195,13 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
                     return;
                 }
                 if (result.image) {
-                    [view pin_updateUIWithImage:result.image animatedImage:nil];
+                    if ([view respondsToSelector:@selector(pin_updateUIWithRemoteImageManagerResult:)]) {
```

Comment:
Minor thought. Could you cache the result of responds to selector in a BOOL? Maybe BOOL updateWithFullResult = [view respondsToSelector:@selector(pin_updateUIWithRemoteImageManagerResult:)];